tests/comparisons/Migration/testCompositeConstraintsSnapshot.php

```diff
@@ -34,26 +34,31 @@ public function up()
             ])
             ->addIndex(
                 [
-                    'slug',
-                ],
-                ['unique' => true]
+                    'category_id',
+                ]
+            )
+            ->addIndex(
+                [
+                    'product_id',
+                ]
```

Comment:
Interesting I would have thought that the constraints would have added the required index. Since that's not the case, this looks good to me.